src/Catalyst.Core.Modules.Ledger.Tests/IntegrationTests/TruffleTest/package.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
@stxphxn I believe we need to add it to gitignore?

Reply:
the package.json? Should be committed along with package-lock above